tests/e2e.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""End to end test for the gcp module."""
+
+from googleapiclient.errors import HttpError
+
+from libcloudforensics import gcp
+
+import unittest
+import logging
+import json
+import time
+
+log = logging.getLogger()
+
+
+class EndToEndTest(unittest.TestCase):
+  """End to end test on GCP.
+  Test that gcp.py is able to correctly make a copy of the boot disk and of another given disk
+  attached to a given instance. The copies are then attached to another instance to run
+  forensics on the disk copies.
+  Add your project information to a project.info file:
```

Comment:
Feel free to add as much space as needed in docstrings, i.e. it's OK to skip a line if it makes things more readable.

Also, where should the `project.info` file be located? Also, if it's a JSON file it's better to call it `project_info.json` or at least give it the `.json` extension.